src/audio/codec_adapter/Kconfig

```diff
@@ -18,4 +18,12 @@ menu "Codec Adapter codecs"
 		  Select for a dummy API codec implementation.
 		  This will cause codec adapter component to include header
 		  files specific to DUMMY base codecs.
+
+	config WAVES_CODEC
+	bool "Waves codec"
+	default n
+	help
+		Select to include Waves codec. Waves codec implements MaxxEffect API.
+		API definition together with pre-compiled library is shared by Waves Audio Ltd.
+		If library is not provided will result in compilation error.
```

Comment:
Do you want to provide a link here so that people can find or contact you for more details.

Reply:
maybe at the top of the c file as well more people asking for processing can never hurt

Reply:
I will check internally regarding this, thank you for the suggestion

Reply:
Added valid contact for product-related questions